manuscript/03.results.md

```diff
@@ -64,12 +64,12 @@ As an application of our model, we used phenotypic measurements of the G1 and S/
+Lapatinib is a EGFR/HER2 inhibitor that induces cell cycle arrest in G1 phase [@doi:10.1358/dot.2006.42.7.985637]. AU565 cells were treated with 25 nM, 50 nM, and 250 nM of lapatinib during the experiment and imaged and tracked for over 96 hours. We fitted the lapatinib treated data to the model with 3 states, based on our BIC-based model selection (Fig. {@fig:expBIC}a). As a result of fitting, the population was classified into two fairly stable states and a cycle (Fig. {@fig:emissionsLPT}a). Lineages that belong to the same state showed that increasing the lapatinib concentration results in the mean of G1 lengths in almost all states, especially in cells of state 1 (Figs. {@fig:emissionsLPT}b-f, @fig:LapatinibTree) and almost no change in the mean of G2 lengths (Fig. {@fig:emissionsLPT}g). The probability of cell death at G1 and G2 phase was higher in state 2 and 3, respectively, but only when the concentration of the drug is more than 50 nM (Figs. {@fig:emissionsLPT}h-i). Further, cells in state 1 seem to be resistant to cell death in all concentrations, and the effect of the drug in these cells is solely on G1 phase extension (Figs. {@fig:emissionsLPT}f-i). 
 
-![**State-specific emissions of the lapatinib-treated data.** (a-d) Lineage trees after fitting the model and state assignment (control, 25 nM, 50 nM, and 250 nM). (e-f) Fit mean time of G1 and G2 phase durations for different concentrations. Most effects of lapatinib treatment is observed in G1 phase. (g-h) Probability of dividing versus dying in the G1 and G2 phase for different concentrations. (i) State transition graph showing the probability of state transitions among the three predicted states.](figure11.svg){#fig:emissionsLPT}
+![**State-specific emissions of the lapatinib-treated data.** (a) State transition graph showing the probability of state transitions among the three predicted states. (b-e) Lineage trees after fitting the model and state assignment (control, 25 nM, 50 nM, and 250 nM). (f-g) Fit mean time of G1 and G2 phase durations for different concentrations. Most effects of lapatinib treatment is observed in G1 phase. (h-i) Probability of dividing versus dying in the G1 and G2 phase for different concentrations.](figure11.svg){#fig:emissionsLPT}
```

Comment:
The results text and captions have not been updated to reflect the new figures. Are you going to do that in a separate branch?

Reply:
Yes, I wanted to do that. I understand it makes more sense if I edit the caption and text here... It is just when the line changes become more than couple of hundreds I feel I should leave more edits to another branch... But in this case, the lines are from svg files so I shouldn't worry about too many changes.
I will edit the text first thing in the morning.

Reply:
No—perfectly fine to do the text changes on a new branch. Let's merge this first.